src/wazuh_db/wdb_delta_event.c

```diff
@@ -228,36 +249,68 @@ STATIC bool wdb_dbsync_stmt_bind_from_json(sqlite3_stmt * stmt, int index, field
                 case cJSON_String: {
                     char * endptr;
                     const int integer_number = (int) strtol(value->valuestring, &endptr, 10);
-                    if (NULL != endptr && '\0' == *endptr &&
-                        SQLITE_OK == sqlite3_bind_int(stmt, index, integer_number)) {
+                    int sqlite3_bind = true;
```

Comment:
It'd have been more correct to use `int sqlite3_bind = SQLITE_ERROR;`, but no changes are requested.

Reply:
Yes, in the middle of the implementation probably I thought of another use for that variable :). Thanks.